pkg/services/control/server/server.go

```diff
@@ -13,6 +14,12 @@ import (
 // Server is an entity that serves
 // Control service on storage node.
 type Server struct {
+	// initialization sync; control service
+	// is the first service to run so no other
```

Comment:
`first service to run` is strange to see on the individual service: this field doesnt "know" what the app does, the app knows how `Service` behaves

Reply:
i can understand you but think that is too much of an abstraction then, do not think the control service can be reused anywhere else, only for the node and only as a main controlling service

> this field doesnt "know" what the app does

yes, but we need this service to be split now. not because it is good for it but because we need it for its caller and this comment would allow a future reader to understand it faster

anyway, fixed the comment